### DISCUSSION TITLE
Full practice Saturday!! Everyone should come.

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#149] [Team Posts]
We would like everyone who can come to attend practice on Saturday. We will be splitting some people off to do some focused work while the rest of us will code-review our Robot Code to date and come up with a TO DO: list of things that are missing or need changes.

Please come well-rested and ready to focus. We have a LOT to cover.

**Comment by coverbeek** [comment]
Please note that master has again been updated with a lot of commits -- you should be sure to pull latest source if you're looking over things.

**Comment by MikasaSucasa** [comment]
Upperclassmen, you will be working separately (with me) so that progress can be made. This will be a very high tempo time doing multiple things (like soldering or code not on your team). You all should be looking ahead to see what has been done thus far so you can be helpful as we sprint towards the finish. Wiring the robot will (hopefully) begin Monday or Tuesday.

**Comment by coverbeek** [comment]
@55QA67 @FallenFlame5567 @JooshO @patrick-philbin -- this means you guys.

If you can, please take a look at what other teams have done so you can pitch in wherever needed. I know you all would like to have a day off, but please take a few minutes tomorrow to review the code and the discussions here on GitHub.
